.github/workflows/ci.yml

```diff
@@ -5,6 +5,12 @@ name: continuous-integration
 
 on: [push, pull_request]
 
+# https://docs.github.com/en/actions/using-jobs/using-concurrency
+concurrency:
```

Comment:
Is not needed here, but it is useful CI workflow configuration. Every time the new commit of a PR is submitted, it will stop the previously running test, this what we usually want and this save the time.